skins/oasis/modules/ContentDisplayController.class.php

```diff
@@ -26,34 +26,34 @@ static function onShowEditLink(&$parser, &$showEditLink) {
 	 */
 	public static function onDoEditSectionLink( $skin, Title $title, $section, $tooltip, &$result, $lang = false ) {
 		global $wgBlankImgUrl, $wgUser;
-		wfProfileIn(__METHOD__);
+		wfProfileIn( __METHOD__ );
 
 		// modify Oasis only (BugId:8444)
-		if (!$skin instanceof SkinOasis) {
-			wfProfileOut(__METHOD__);
+		if ( !$skin instanceof SkinOasis ) {
+			wfProfileOut( __METHOD__ );
 			return true;
 		}
 
 		$result = ''; // reset result first
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)